tests/chat/test_reasoning.py

```
@@ -0,0 +1,239 @@
+import logging
+from unittest.mock import MagicMock, patch
```

Ruff (0.11.9)

`unittest.mock.patch` imported but unused

Remove unused import: `unittest.mock.patch`

(F401)